alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -143,6 +145,66 @@ private function appendUiConfToContent($content)
 
 		return $content;
 	}
+
+	private function appendEnvConfigToContent($content)
+	{
+
+		$protocol = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? "https" : "http";
```

Comment:
Use requestUtils::getRequestProtocol() it does the same thing.

Reply:
@yossipapi agree, but I think this should be also abstracted to infraUtils or statUtils so any service will be able to resolve these urls the same way.
@erankor - do you have plans to expose the stats end points in some infra and via INI config? maybe you already do?